WIP: Issue/5 todo

Pergamene opened the pull request.

Commit:
just some starter notes

Austin-Glenn committed:
Merge branch 'master' of https://github.com/Pergamene/project-spiderweb-service into issue/5-todo

Austin-Glenn committed:
changed file path for sql

Austin-Glenn committed:
added files needed to resolve all @issue

Austin-Glenn committed:
Merge branch 'master' of https://github.com/Pergamene/project-spiderweb-service into issue/5-todo

Austin-Glenn committed:
wip to review.

Commit:
should work, haven't tested with actually unmarshalling the request,
but that should be done as part of the pagedetail handler test anyway.

Pergamene committed:
Merge pull request #8 from Pergamene/issue/5-todo-partitionmarshel

should work, haven't tested with actually unmarshalling the request,

Austin-Glenn committed:
removed some comments and fixed a typo on a method call

Austin-Glenn committed:
Merge branch 'master' of https://github.com/Pergamene/project-spiderweb-service into issue/5-todo

Austin-Glenn committed:
Merge branch 'master' of https://github.com/Pergamene/project-spiderweb-service into issue/5-todo

Austin-Glenn committed:
Merge branch 'master' of https://github.com/Pergamene/project-spiderweb-service into issue/5-todo

rhyeen commented:
@Austin-Glenn what's left on this?  Looks like you really just need to bind the service to the data store.

rhyeen commented on internal/services/pagedetail/service.go:
```diff
@@ -0,0 +1,30 @@
+	// err = s.PageDetailStore.UpdatePageDetail(params.Detail)
+	// if err != nil {
+	// 	return errors.Wrapf(err, "failed to update page detail: %+v", params)
+	// }
```
Here

Austin-Glenn committed:
possibly finished, after conflicts resolved.

Austin-Glenn committed:
fixed merge conflicts

Merge branch 'master' of https://github.com/Pergamene/project-spiderweb-service into issue/5-todo

# Conflicts:
#	cmd/server/main.go
#	internal/services/page/service.go
#	internal/stores/mysqlstore/pagestore.go

rhyeen commented on cmd/server/main.go:
```diff
@@ -97,11 +99,15 @@ func setupHandler(apiPath, staticPath, datacenter string, mysqldb *sql.DB) (http
 		VersionStore:      versionStore,
 		UserStore:         userStore,
 	}
+	pageDetailService := pagedetailservice.PageDetailService{
```
^ `pageDetailStore` doesn't exist.  Your fix of merge conflicts wiped it.
